### ISSUE TITLE
svg file return from saveSVG() is invalid (Internet Explorer 11)

### ISSUE THREAD

**Issue by icaha** [bpmn-io/bpmn-js#594]
Source code like below ...

```
var BpmnModeler = require('bpmn-js/lib/Modeler');
...
bpmnModeler.saveSVG(done);
```

When the code is executed on Internet Explorer (v11), saved svg file is ...

```
...
&lt;g xmlns="http://www.w3.org/2000/svg" class="djs-group" xmlns="http://www.w3.org/2000/svg"&gt;
...
```

The xmlns attribute happens one more time in each g element.
When it is opened from Chrome, error message occurs like this...

This page contains the following errors:
error on line 4 at column 2106: Attribute xmlns redefined

**Comment by vkatushenok** [comment]
Here is an example temporary workaround:

```javascript
var rx = /<g xmlns=[^>]*/g;
svg = svg.replace(rx, '<g');
```

N.B. This would remove unnecessary exported classes, as well.

**Comment by romero83** [comment]
The above workaround not worked for me but still good catch.
I modified it to make it work:
```
var rx = /<g xmlns="[^ ]*/g;
svg = svg.replace(rx, '<g');
```

**Comment by nikku** [comment]
Closed via #715.
